<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#80ffef;
}</string>
   </property>
   <property name="text">
    <string>用户注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="information">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:yellow;
}</string>
   </property>
   <property name="text">
    <string>注：请按照用户真实信息填写，信息错误将无法通过验证</string>
   </property>
  </widget>
  <widget class="ElaComboBox" name="identity_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>学生</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教师</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="identity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>身份：</string>
   </property>
  </widget>
  <widget class="QLabel" name="account">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>学号/工号：</string>
   </property>
  </widget>
  <widget class="ElaLineEdit" name="account_edit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background: transparent; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="ElaLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background: transparent; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="ElaComboBox" name="sex_box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="college">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>学院：</string>
   </property>
  </widget>
  <widget class="QLabel" name="profession">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>专业：</string>
   </property>
  </widget>
  <widget class="ElaComboBox" name="college_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <item>
    <property name="text">
     <string>计算机科学与技术学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>其他学院</string>
    </property>
   </item>
  </widget>
  <widget class="ElaComboBox" name="profession_box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <item>
    <property name="text">
     <string>计算机科学与技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>软件工程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>信息安全</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="ElaLineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background: transparent; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="ElaLineEdit" name="confirm_edit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit { background: transparent; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="confirm">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QtMaterialFlatButton" name="register_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>确认注册</string>
   </property>
  </widget>
  <widget class="QToolButton" name="min_btn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialflatbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ElaComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">elacombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ElaLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">elalineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
